can_msgs/Frame msg
string node_name
---
string ack